roles/basic_app_examples/cake_php_backup_restore/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+# Deploy cakephp
+- name: Create a namespace
+  shell: "oc new-project cakephp-example"
+
+- name: Deploy cakephp app
+  shell: "oc new-app -f https://raw.githubusercontent.com/sclorg/cakephp-ex/master/openshift/templates/cakephp.json"
```

Comment:
Good test case, might be worth mentioning in the readme that S2I is currently unsupported